lenses_python/ProcessorHandler.py

```diff
@@ -45,12 +45,11 @@ def CreateProcessor(self, name, sql, runners, clusterName, namespace, pipeline):
                 clusterName=clusterName
             )
 
-
         headers = {'Content-Type': 'application/json', 'Accept': 'text/plain',
-                    'x-kafka-lenses-token': self.token}
+                   'x-kafka-lenses-token': self.token}
```

Comment:
same format as above.

Reply:
Moved the dictionary format to the what was suggested